src/lib/styles/material/_tree.scss

```diff
@@ -0,0 +1,119 @@
+.mat-tree {
+    height: calc(100% - var(--height-lg));
+
+    &.collapsed {
+        .mat-tree-node {
+            border: 0;
+
+            a.drawer-link .node-label,
+            .mat-icon-button.btn-close,
+            .mat-icon-button.btn-caret,
+            &.mat-tree-node-level-1,
+            &.mat-tree-node-level-2,
+            &.mat-tree-node-level-3,
+            &.mat-tree-node-level-4,
+            &.mat-tree-node-level-5 {
+                display: none;
+            }
+
+            &.mat-tree-node-level-1 + .mat-tree-node-level-0,
+            &.mat-tree-node-level-0 + .mat-tree-node-level-1 {
+                margin-top: 0;
+            }
+
+            &.mat-tree-node-level-0 {
+                border-top: none;
+            }
+        }
+    }
+
+    .mat-tree-node {
+        min-height: initial;
+        height: var(--height-lg);
+        color: var(--color-structure-9);
+        font-size: var(--font-size-6);
+        padding-right: var(--space-sm);
+
+        &.mat-tree-node-level-0 {
+            border-bottom: 1px solid var(--color-structure-3);
+            border-top: 1px solid var(--color-structure-3);
+            padding-left: var(--space-xl) !important;
+        }
+
+        &.mat-tree-node-level-0 + .mat-tree-node-level-0,
+        &:first-child {
+            border-top: none;
+        }
+
+        &.mat-tree-node-level-1 + .mat-tree-node-level-0,
+        &.mat-tree-node-level-0 + .mat-tree-node-level-1 {
+            margin-top: var(--space-md);
+        }
+
+        &.mat-tree-node-level-1 {
+            padding-left: var(--height-lg) !important;
+        }
+
+        &.mat-tree-node-level-2 {
+            padding-left: calc(2 * var(--height-lg) - var(--space-xl)) !important;
+        }
+
+        &.mat-tree-node-level-3 {
+            padding-left: calc(3 * var(--height-lg) - var(--space-xl)) !important;
+        }
+
+        &.mat-tree-node-level-4 {
+            padding-left: calc(4 * var(--height-lg) - var(--space-xl)) !important;
+        }
+
+        &.mat-tree-node-level-5 {
+            padding-left: calc(5 * var(--height-lg) - var(--space-xl)) !important;
+        }
+
+        .node-icon {
+            padding-right: var(--space-xl);
+        }
+
+        a.drawer-link {
+            color: inherit;
+            cursor: pointer;
+            display: flex;
+            align-items: center;
+            margin-right: auto;
+
+            &.active {
+                color: var(--color-active);
+            }
+
+            .node-icon {
+                font-size: 18px;
+            }
+
+            .node-label {
+                font-size: var(--font-size-2);
+            }
+        }
```

Comment:
Das sind doch alles styles für unseren speziellen Anwendungsfall in der Sidenav. Diese Klassen existieren in nem normalen Tree doch garnicht!?